Onboarding/README.md

```diff
@@ -3,40 +3,45 @@
 ![Onboarding Image](Figures/Logo.png "CanDIG Logo")
 
 - [CanDIG Project Onboarding](#candig-project-onboarding)
-    - [The Canadian Distributed Infrastructure for Genomics](#the-canadian-distributed-infrastructure-for-genomics)
+  - [The Canadian Distributed Infrastructure for Genomics](#the-canadian-distributed-infrastructure-for-genomics)
 - [Project Communications](#project-communications)
-    - [Tools](#tools)
-    - [Meetings](#meetings)
+  - [Tools](#tools)
+  - [Meetings](#meetings)
 - [Project Background](#project-background)
-    - [CanDIG Project Overview](#candig-project-overview)
-        - [What Success Looks Like](#what-success-looks-like)
-        - [Technical Principles](#technical-principles)
-    - [The CanDIG Ecosystem](#the-candig-ecosystem)
-        - [Entities Involved In CanDIG](#entities-involved-in-candig)
-        - [External Partners and Other Projects](#external-partners-and-other-projects)
-            - [GA4GH](#ga4gh)
-            - [CHORD](#chord)
-            - [CINECA](#cineca)
-            - [GenAP](#genap)
-            - [PROFYLE](#profyle)
-            - [TF4CN](#tf4cn)
-    - [CanDIG Status](#candig-status)
+  - [CanDIG Project Overview](#candig-project-overview)
+    - [What Success Looks Like](#what-success-looks-like)
+    - [Technical Principles](#technical-principles)
+  - [The CanDIG Ecosystem](#the-candig-ecosystem)
+    - [Entities Involved In CanDIG](#entities-involved-in-candig)
+    - [External Partners and Other Projects](#external-partners-and-other-projects)
+      - [GA4GH](#ga4gh)
```

Comment:
I am a bit confused by this file. It has a lot of changes for indentation. Is that on purpose? Just curious.

Reply:
Honestly this file is magic to me. I made the change from 4:00 -> 2:00 and when I saved the table of contents seemed to auto-update somehow